### PR TITLE
VideoInfo support

### PR DESCRIPTION
**Pull request by broilogabriel**
Created class for VideoInfo, Variant and included attribute in the Media for issue #181. Fixtures files updated to support the new field and one new json included in home_timeline.json to test the new parser.

This PR is based in the tag v5.2.

### CODE REVIEW

File: src/test/resources/fixtures/rest/statuses/home_timeline.json [hunk 1]
```diff
@@ -1819,7 +1822,21 @@
           "source_status_id": null,
           "source_status_id_str": null,
           "type": "animated_gif",
-          "url": "http://t.co/zytXLkyndk"
+          "url": "http://t.co/zytXLkyndk",
+          "video_info": {
```
Comment (DanielaSfregola) [inline src/test/resources/fixtures/rest/statuses/home_timeline.json]: Nice!

**Comment by DanielaSfregola** [comment]
It may take us longer than expected to get to 6.0....let me park those commit on a separate branch so that we can release 5.3 ...

**Comment by broilogabriel** [comment]
That's cool @DanielaSfregola I'm really looking forward for the next release, for now I'm using my fork because this change is mandatory for me.

**Comment by DanielaSfregola** [comment]
@broilogabriel twitter4s 5.3 is on its way to maven central ;)

**Comment by broilogabriel** [comment]
Amazing! Thanks @DanielaSfregola, I'll definitely try to contribute more to this project.

**Pull request merged**